eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -63,6 +63,9 @@ parameters:
 
 stages:
   - stage: Build
+    variables:
+      - name: BuildTargetingStringValue
+        value: $[ coalesce(variables['BuildTargetingString'], '${{ parameters.BuildTargetingString }}') ]
```

Comment:
This approach will work but only if BuildTargetingString isn't set as a variable anywhere else.

Reply:
Yep.